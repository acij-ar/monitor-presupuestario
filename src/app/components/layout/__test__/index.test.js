const React = require('react');
const { shallow } = require('enzyme');

const Layout = require('..');

describe('Layout component', () => {
  it('should match snapshot', () => {
    const componentProps = { mocked: 'children-props' };
    const props = {
      scripts: ['http://script.js'],
      styles: ['http://style.css'],
      componentProps,
    };
    const wrapper = shallow(
      <Layout {...props} >
        <div id="dummy-children" {...componentProps} />
      </Layout>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
