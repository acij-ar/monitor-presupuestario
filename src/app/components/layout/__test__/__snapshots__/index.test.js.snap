// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should match snapshot 1`] = `
<html
  lang="es-ar"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      content="Monitor presupuestario es una plataforma en la que encontrarás información de los diferentes organismos, ministerios, secretarías, programas y actividades presupuestarias desde el año 2007 hasta la actualidad y podrás monitorear la asignación y ejecución del presupuesto naciona"
      name="description"
    />
    <title>
      Monitor presupuestario
    </title>
    <link
      href="/static/favicon.ico"
      rel="icon"
      type="image/x-icon"
    />
    <link
      href="/static/http://style.css"
      key="http://style.css"
      rel="stylesheet"
    />
    <link
      href="https://fonts.googleapis.com/css?family=Rubik:300,400&display=swap"
      rel="stylesheet"
    />
  </head>
  <body>
    <div
      id="root"
    >
      <div
        id="dummy-children"
        mocked="children-props"
      />
    </div>
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__INITIAL__DATA__={\\"mocked\\":\\"children-props\\"};",
        }
      }
    />
    <script
      key="http://script.js"
      src="/static/http://script.js"
    />
  </body>
</html>
`;
